### ISSUE TITLE
Title is now set to class in citrix

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#6435]
### Bug or Regression?

Regression

### System Info and Version

Since verison 41 my citrix session now has the title 

```
Window 602fb9a46640 -> wfica:
	class: Wfica
	title: wfica
	initialClass: Wfica
	initialTitle: wfica
```

### Description

Before it had an appropriate title to the window it was, something changed.

### How to reproduce

launch citrix workspace on the new version.

**Comment by MightyPlaza** [comment]
Are you sure it was not a program update itself?
does downgrading hyprland fix it?

**Comment by Kommynct** [comment]
Yup, downgrading changes it back to it's normal title 


Window 564b3a931560 -> Office 365 Desktop (SSL/TLS Secured, 256 bit):
	mapped: 1
	hidden: 0
	at: 3840,0
	size: 1920,1060
	workspace: 3 (3: )
	floating: 0
	monitor: 1
	class: Wfica
	title: Office 365 Desktop (SSL/TLS Secured, 256 bit)
	initialClass: Wfica
	initialTitle: Office 365 Desktop (SSL/TLS Secured, 256 bit)
	pid: 286861
	xwayland: 1
	pinned: 0
	fullscreen: 0
	fullscreenmode: 0
	fakefullscreen: 0
	grouped: 0
	swallowing: 0
	focusHistoryID: 1

**Comment by MightyPlaza** [comment]
can you bisect?
(it's prolly the wayland rewrite)

**Comment by Kommynct** [comment]
Is there a guide you recommend for this? I have no idea what I'm doing.

**Comment by MightyPlaza** [comment]
> Is there a guide you recommend for this? I have no idea what I'm doing.

https://wiki.hyprland.org/Crashes-and-Bugs/#bisecting-an-issue

**Comment by jprostko** [comment]
After doing some test builds of Hyprland, and starting Citrix Workspace, I confirmed that this was fixed by a95df6b57e68785c13fd2d7f93d9eba28b9406ed.  The title and initialTitle now display correctly.

This issue can be closed.

**Comment by Kommynct** [comment]
Yup, it's fixed.
